### ISSUE TITLE
Passing working directory parameter through REPOSILITE environment variable fails

### ISSUE THREAD

**Issue by guiliguili** [dzikoysk/reposilite#1437]
### What happened?

Passing working directory parameter through REPOSILITE environment variable fails :

```bash
$ docker run -it -p 8080:8080 -v /path/to/home:/home -e REPOSILITE_OPTS='--working-directory=/home' dzikoysk/reposilite:3.0.0-rc.2
```

### Reposilite version

3.x

### Relevant log output

```shell
/Users/i303934/SAPDevelop/Products/reposilite/git/reposilite/home:/home -e REPOSILITE_OPTS='--working-directory=/home' dzikoysk/reposilite:3.0.0-rc.2
option '--working-directory' (<workingDirectoryName>) should be specified only once
Usage: reposilite [-dv] [--help] [-gc=<configurationRequested>] [-h=<hostname>]
                  [-lc=<localConfigurationFile>]
                  [-lcm=<localConfigurationMode>] [--level=<level>] [-p=<port>]
                  [-sc=<sharedConfigurationFile>] [-wd=<workingDirectoryName>]
                  [-t=<tokenEntries>]...
  -d, --debug, --test-env   Enable test mode
      -gc, --generate-configuration=<configurationRequested>
                            Generate default template of the configuration
                              file. Supported templates:
                            configuration.cdn - Local configuration file
                            configuration.shared.json - Shared configuration
                              file
  -h, --hostname=<hostname> Override hostname from configuration
      --help                Display help message
      -lc, --local-config, --local-configuration=<localConfigurationFile>
                            Set custom location of local configuration file
      -lcm, --local-config-mode, 
        --local-configuration-mode=<localConfigurationMode>
                            Supported local configuration modes:
                            auto - process and override main configuration file
                            none - disable automatic updates of configuration
                              file
      --level, --channel=<level>
                            Default logging channel
  -p, --port=<port>         Override port from configuration
      -sc, --shared-config, --shared-configuration=<sharedConfigurationFile>
                            Set custom location of shared configuration file
  -t, --token=<tokenEntries>
                            Create temporary token with the given credentials
                              in name:secret format
                            Created token has all permissions
  -v, --version             Display current version of reposilite
      -wd, --working-directory=<workingDirectoryName>
                            Set custom working directory of application instance
Exception in thread "main" kotlin.UninitializedPropertyAccessException: lateinit property workingDirectory has not been initialized
	at com.reposilite.ReposiliteParameters.getWorkingDirectory(ReposiliteParameters.kt:42)
	at com.reposilite.configuration.local.LocalConfigurationFactory.createLocalConfiguration(LocalConfigurationFactory.kt:28)
	at com.reposilite.ReposiliteFactory.createReposilite(ReposiliteFactory.kt:38)
	at com.reposilite.ReposiliteFactory.createReposilite(ReposiliteFactory.kt:35)
	at com.reposilite.ReposiliteLauncherKt.createWithParameters(ReposiliteLauncher.kt:41)
	at com.reposilite.ReposiliteLauncherKt.main(ReposiliteLauncher.kt:22)
```

**Comment by dzikoysk** [comment]
That's because the `workingDirectory`:

* https://github.com/dzikoysk/reposilite/blob/main/Dockerfile#L31
* https://github.com/dzikoysk/reposilite/blob/main/entrypoint.sh#L14

I'm not sure if it makes sense to expose this property 🤔

**Comment by guiliguili** [comment]
Let me give you some more context. I am trying to deployment reposilite to Azure Web App Service. 
I tried to configure external volumes in Azure App Service, but due the following limitation, it cannot be used with sqlite : https://github.com/MicrosoftDocs/azure-docs/issues/47130
So the only I have right now is to use Use persistent shared storage in [Azure App Service](https://docs.microsoft.com/en-us/azure/app-service/configure-custom-container?pivots=container-linux) which makes the `/home`directory persistent. That's why I need be able to specify a different working directory.

I have created the following PR to fix the issue : https://github.com/dzikoysk/reposilite/pull/1438
